v0.5.5 fails to build with GHC 8.10.1

```
src/Test/QuickCheck/Checkers.hs:256:30: error:
    Ambiguous occurrence ‘First’
    It could refer to
       either ‘Data.Monoid.First’,
              imported from ‘Data.Monoid’ at src/Test/QuickCheck/Checkers.hs:(53,1)-(57,27)
           or ‘Data.Semigroup.First’,
              imported from ‘Data.Semigroup’ at src/Test/QuickCheck/Checkers.hs:69:1-21
    |
256 | instance EqProp a => EqProp (First a)
    |                              ^^^^^

src/Test/QuickCheck/Checkers.hs:257:30: error:
    Ambiguous occurrence ‘Last’
    It could refer to
       either ‘Data.Monoid.Last’,
              imported from ‘Data.Monoid’ at src/Test/QuickCheck/Checkers.hs:(53,1)-(57,27)
           or ‘Data.Semigroup.Last’,
              imported from ‘Data.Semigroup’ at src/Test/QuickCheck/Checkers.hs:69:1-21
    |
257 | instance EqProp a => EqProp (Last a)
    |                              ^^^^
```

I think the issue was introduced in https://github.com/conal/checkers/commit/e7bb6fcbe4c1440131b7306a8a45ab8a1210e7ce.

@isovector Could you possibly take a look?

--- a/src/Test/QuickCheck/Checkers.hs
+++ b/src/Test/QuickCheck/Checkers.hs
@@ -50,11 +50,7 @@ import Control.Arrow ((***),first)
 import qualified Control.Exception as Ex
 import Data.List (foldl')
 import Data.List.NonEmpty (NonEmpty (..))
-import Data.Monoid hiding (
-#if __GLASGOW_HASKELL__ < 810
-    First, Last
-#endif
-                          )
+import Data.Monoid hiding (First, Last)
 
 import Data.Complex
 import Data.Proxy

